app/models/crosswalk_issue.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CrosswalkIssue < ApplicationRecord
+  belongs_to :crosswalk
+  belongs_to :ipeds_hd
+  belongs_to :weam
+
+  # rubocop:disable Metrics/MethodLength
+  def self.rebuild
+    sql = <<-SQL
+      INSERT INTO crosswalk_issues (
+        weam_id,
+        crosswalk_id,
+        ipeds_hd_id
+      )
+      SELECT
+        weams.id,
+        crosswalks.id,
+        ipeds_hds.id
+      FROM weams
+        LEFT OUTER JOIN ipeds_hds ON  weams.cross = ipeds_hds.cross
+          OR weams.ope = ipeds_hds.ope
+        LEFT OUTER JOIN crosswalks ON weams.facility_code = crosswalks.facility_code
+      WHERE
+        (institution_of_higher_learning_indicator = true OR non_college_degree_indicator = true)
+        AND NOT(
+          weams.cross IS NOT NULL
+          AND ipeds_hds.cross IS NOT NULL
+          AND crosswalks.cross IS NOT NULL
+          AND weams.cross = ipeds_hds.cross
+          AND weams.cross = crosswalks.cross
+        )
```

Comment:
In English, this `AND NOT` removes all results where `weams.cross` is populated AND `ipeds_hds.cross` is populated AND `crosswalks.cross ` is populated AND all three values are the same.  ALL of those conditions have to be true.

What about the scenario where  your LEFT OUTER JOINS joined on no `ipeds_hds` or `crosswalks` records.  In that case you'd have a `null` `ipeds_hds.cross` and `null` `crosswalks.cross`, and those results would stay included.  Your `crosswalks_issues` table would end up with a record that had a `weams_id` populated but a `null` `crosswalks_id` and `null` `ipeds_hds_id`.
Is that OK?

Or do you want:
```SQL
AND NOT(
          weams.cross IS NOT NULL
          AND weams.cross = ipeds_hds.cross
          AND weams.cross = crosswalks.cross
        )
```

Reply:
Yes, this is what they've asked for because it's an "issue" they'd need to resolve.